src/src_V2/components/tooltip/TooltipText.jsx

```diff
@@ -0,0 +1,23 @@
+import { useRef } from 'react';
+import { Tooltip } from '@mui/material';
+
+import { useHoverTooltip } from '../../hooks';
+import styles from './TooltipText.module.css';
+
+export default function TooltipText({ text, maxWidth }) {
+  const textElementRef = useRef();
+
+  const hoverStatus = useHoverTooltip(text, textElementRef);
+
+  return (
+    <Tooltip title={text} interactive="true" disableHoverListener={!hoverStatus}>
+      <div
+        ref={textElementRef}
+        className={styles.tooltip}
+        style={{ maxWidth: maxWidth || '100px' }}
```

Comment:
pls avoid using inline styles and use Box instead div

Reply:
In this case, inline style can't be avoided. It's being applied dynamically (will make Box and sx={})

Reply:
yep, pls use mui styling solution there.

Reply:
> In this case, inline style can't be avoided. It's being applied dynamically (will make Box and sx={})

@aninaslyan jan just for dinamic styling, you also can dinamicly add className and remove so with this approach we can avoid using inline ones.